chainer/links/normalization/batch_normalization.py

```diff
@@ -59,14 +60,19 @@ class BatchNormalization(link.Link):
 
     """
     def __init__(self, size, decay=0.9, eps=1e-5, dtype=numpy.float32,
-                 use_gamma=True, use_beta=True):
+                 use_gamma=True, use_beta=True,
+                 initial_gamma=None, initial_beta=None):
         super(BatchNormalization, self).__init__()
         if use_gamma:
             self.add_param('gamma', size, dtype=dtype)
-            self.gamma.data.fill(1)
+            if initial_gamma is None:
+                initial_gamma = numpy.ones(size)
```

Comment:
Use `chainer.Initializers.One` instead of `ndarray` because `init_weight` implicitly converts `ndarray` to `Initializer` [here](https://github.com/pfnet/chainer/blob/master/chainer/initializers/__init__.py#L46).